src/daemon/dbus_service.h

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2019 Luxoft Sweden AB
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#ifndef TEMPLATE_DBUS_SERVICE_DAEMON_DBUS_SERVICE_H
+#define TEMPLATE_DBUS_SERVICE_DAEMON_DBUS_SERVICE_H
+
+#include <glibmm.h>
+
+#include "generated/dbus/template_dbus_service_common.h"
+#include "generated/dbus/template_dbus_service_stub.h"
+
+namespace TemplateDBusService::Daemon
+{
+    class DBusService final : private com::luxoft::TemplateDBusServiceStub
+    {
+    public:
+        explicit DBusService(const Glib::RefPtr<Glib::MainLoop> &main_loop);
+        ~DBusService() final;
+
+        DBusService(const DBusService &other) = delete;
+        DBusService(DBusService &&other) = delete;
+        DBusService &operator=(const DBusService &other) = delete;
+        DBusService &operator=(DBusService &&other) = delete;
+
+        void own_name();
```

Comment:
Then this may be named acquire_name() ?

Reply:
I would rather keep `own` since it is "D-Bus terminology". But I don't know. Does it matter that much? It is not like `own` does not go together as the opposite for `release`.

Reply:
And vice versa.

Reply:
But, hum... the mix it in GDBus. OK, whatever, I'll change it. :)

Reply:
One own's something after acquiring it. Own itself is not the process of acquiring. Is unown also dbus term?

Reply:
@TSAnsari Did you remove a comment here? Anyway, they talk more about "owning a name" in the D-Bus repository than "acquiring a name".

Will probably revert both fixup commits related to this on Monday. It does not matter that much but probably just confusing to call it something else when the Glib::GDBus functions that are invoked are called `own_name()` and `unown_name()`.

Reply:
I don't mind if you use the glib standard names own_name and unown_name. Just didn't liked mixing them, like own and release.

Reply:
Reverted commits that changed name. Resolving.